.dmypy.json

```diff
@@ -0,0 +1 @@
+{"pid": 125392, "connection_name": "\\\\.\\pipe\\dmypy-cigaVDFh.pipe"}
```

Comment:
Is there any reason for this to be in git?
If not, please remove from the PR and add a line in .gitignore instead.

Reply:
added to .gitignore and git rm